app/controllers/event_notes_controller.rb

```diff
@@ -3,7 +3,7 @@ class EventNotesController < ApplicationController
 
   rescue_from Exceptions::EducatorNotAuthorized, with: :redirect_unauthorized!
 
-  before_action :authorize!, except: [:names]
+  before_action :authorize!, except: [:names] # TODO: Why is this here? :names is an action called on students only, right?
```

Comment:
You are correct @erose, the `except` clause doesn't belong here. There is no `:names` action for `EventNotesController`, so `except: [:names]` does nothing in this context.